changes/ocd-3563.sql

```diff
@@ -0,0 +1,18 @@
+DROP TABLE IF EXISTS openchpl.listing_validation_report;
+
+CREATE TABLE openchpl.listing_validation_report (
+  id                        bigserial NOT NULL,
+  chpl_product_number       text NOT NULL,
```

Comment:
What do you think about storing this in the table even though it can change with listing updates? Would it be better to just use the certified product id as a foreign key? I'm asking because maybe it doesn't matter but it depends how often the job is going to run to calculate the errors. If it runs daily then maybe it's okay to store this string but if it's less frequently then I feel like we could run into inconsistencies with the chpl product number changing over time.